src/pages/getting-started/app-guides/in-app-upgrades/index.mdx

```diff
@@ -114,6 +114,35 @@ The **Summary** column shows the unit price of an upgrade as well as the total p
 
 Users can select the resource quantity in the **Confirm and pay** modal.
 
+Additionally, for fullscreen widgets, LiveChat app provides a way to embed the table with upgrades in a modal. You can utilize this invoking `window.postMessage()` method with `event_name: 'open_in_app_upgrades_modal'` parameter.
+
+<Code>
+<CodeSample path={'OPEN IN-APP UPGRADES MODAL'}>
+
+```js
+const openInAppUpgrades = () => {
+  const message = JSON.stringify({
+    event_name: "open_in_app_upgrades_modal",
+    event_data: null,
```

Comment:
What's event_data for?

Reply:
actually, in this case it's for nothing (at least for now) so I'll just delete it - leftover from similar types of events, but we don't need it here